code/game/objects/structures/window.dm

```diff
@@ -190,6 +190,16 @@
 				to_chat(user, "<span class='notice'>You pry the window into the frame.</span>")
 			return
 
+	if(istype(I, /obj/item/pickaxe/drill/jackhammer))
+		to_chat(user, "<span class='notice'>You begin to smash though [src]...</span>")
+		if(do_after(user, 75, target = src))
+			if(!istype(src, /obj/structure/window))
+				return TRUE
```

Comment:
jesse

Reply:
This is behavior inherited from other jackhammer smash procs I ported from tg, you'll see they have a similar define. I think it prevents a bug related to triggering the effect on other objects after clicking a window?
Not entirely sure, but I made sure to preserve it.